install/install.sh

```diff
@@ -82,6 +82,19 @@ fetch() {
   fi
 }
 
+install() {
+ local sudo 
+ if [ -w $BIN_DIR ]; then
+	 sudo=''
+ else
+	 info "Root privileges are required to proceed the installation"
```

Comment:
I worry that the important visual queue that user interaction is needed isn't very clear with `info`:

![image](https://user-images.githubusercontent.com/4658208/73969837-1b8d1780-48ea-11ea-9494-f80810cac2e7.png)

What are your thoughts on using a blue exclamation mark for "input required"?
![image](https://user-images.githubusercontent.com/4658208/73969954-51320080-48ea-11ea-8513-6eee64ed2a97.png)

Reply:
Perhaps we should use the opportunity to explain why root privileges are needed, specifying that it's because they don't have write access to that directory:

![image](https://user-images.githubusercontent.com/4658208/73970875-ef729600-48eb-11ea-8398-4c65ededae02.png)

Reply:
Good catcha! I'll chagne to warn instead info